contrib/CMakeLists.txt

```diff
@@ -7,6 +7,10 @@ set(PYVXL_CONTRIB_MAKE_ALL FALSE CACHE BOOL "Build every optional vxl contrib mo
 # INDIVIDUAL GUARDS, inherit from ALL GUARD
 set(PYVXL_CONTRIB_MAKE_BPGL ${PYVXL_CONTRIB_MAKE_ALL} CACHE BOOL "Turn on BPGL build")
 set(PYVXL_CONTRIB_MAKE_BRAD ${PYVXL_CONTRIB_MAKE_ALL} CACHE BOOL "Turn on BRAD build")
+set(PYVXL_CONTRIB_MAKE_BRIP ${PYVXL_CONTRIB_MAKE_ALL} CACHE BOOL "Turn on BRIP build")
+set(PYVXL_CONTRIB_MAKE_BVGL ${PYVXL_CONTRIB_MAKE_ALL} CACHE BOOL "Turn on BVGL build")
```

Comment:
Is this used?

Reply:
BRIP is used. The BVGL bindings weren't needed so I never ended up committing them. I've removed the reference to BVGL.